app/models/template.rb

```diff
@@ -299,6 +299,10 @@ def remove_trailing_chars
     self.template = template.tr("\r", '') if template.present?
   end
 
+  def run_template_changes_validation?
+    (self.locked? || self.locked_changed?) && self.persisted? && !ForemanSeeder.is_seeding
```

Comment:
self is redundant here, if you will rebase on develop rubocop will fail

Reply:
done